packages/plugins/typescript/react-apollo/src/visitor.ts

```diff
@@ -68,7 +68,10 @@ export class ReactApolloVisitor extends ClientSideBaseVisitor<ReactApolloRawPlug
   ${operationResultType},
   ${operationVariablesTypes},
   ${propsTypeName}<TChildProps>>) {
-    return ReactApollo.with${titleCase(node.operation)}<TProps, ${operationResultType}, ${operationVariablesTypes}, ${propsTypeName}<TChildProps>>(${documentVariableName}, operationOptions);
+    return ReactApollo.with${titleCase(node.operation)}<TProps, ${operationResultType}, ${operationVariablesTypes}, ${propsTypeName}<TChildProps>>(${documentVariableName}, {
+      alias: 'with${operationName}',
```

Comment:
Can you please check if `operationName` is set before using it? Because it might be empty

Reply:
Do you ask for only here or in other parts as well?
because we don't check for it in anywhere in this plugin.

Reply:
Hmm so let's leave it as is. maybe later we can add it on a higher level.